StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/build/StyleCop.Analyzers.targets

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Attempt to automatically include stylecop.json in the <AdditionalFiles /> item group.
+    
+    To disable this functionality, set the MSBuild property "AutoIncludeStyleCopJson" to "false".  Example:
+    
+    <PropertyGroup>
+      <AutoIncludeStyleCopJson>false</AutoIncludeStyleCopJson>
+    </PropertyGroup>
+  -->
+  <ImportGroup Condition=" '$(AutoIncludeStyleCopJson)' != 'false' ">
```

Comment:
:memo: Might need to move this condition to the `<ItemGroup>` elements in the imported files since the import group will be evaluated before the properties in the main project file take effect.